### ISSUE TITLE
Continuously support new OCaml versions

### ISSUE THREAD

**Issue by CraigFe** [ocurrent/ocaml-ci#95]
We've been statically tracking support for the `4.0x` compiler versions in this repository (https://github.com/ocurrent/ocaml-ci/pull/47, https://github.com/ocurrent/ocaml-ci/pull/53, https://github.com/ocurrent/ocaml-ci/pull/66, https://github.com/ocurrent/ocaml-ci/pull/90). 

This information should probably be derived from [`ocaml-version`](https://github.com/ocurrent/ocaml-version), so that we don't have to manually update it each time.

**Comment by avsm** [comment]
This is mostly done in #236 (except of course we have to manually bump the ocaml-version package, but that's fine).

**Comment by talex5** [comment]
This was fIxed in #241. e.g.

```ocaml
let default_compiler = OV.(Releases.latest |> without_patch)
```
